data_io/connection.py

```diff
@@ -211,15 +211,18 @@ def build(self, url: str) -> BaseConnection:
             """
 
             # Get credentials
-            username, password = self.get_credentials(url)
+            username, password, user = self.get_credentials(url)
 
             # Alter location parts to create a new URL
             # Ran into URL parsing issues, so needed to
             # quote username and password
             _url = (
                 url
-                .replace(
+                .replace( dev_url
+                    f'{user}@',
```

Comment:
`replace` is incorrectly called here - does this run?

Reply:
There was a merge conflict. looks like when I resolved it I forgot to delete `dev_url`

Reply:
Yes, I think this will work.